sys/arm64/include/pmap.h

```diff
@@ -188,7 +188,9 @@ extern void (*pmap_invalidate_vpipt_icache)(void);
 static inline int
 pmap_vmspace_copy(pmap_t dst_pmap __unused, pmap_t src_pmap __unused)
```

Comment:
Do the `__unused` do anything but suppress warnings?

Reply:
Just that (contrast with __used, which tells the compiler things it thinks are unused are in fact used but in ways it can't see, i.e. written in assembly without using extended asm operands)